src/easylink/runner.py

```diff
@@ -174,18 +202,41 @@ def write(self, text: str) -> int:
                 if line.strip():
                     filtered_line = _filter_snakemake_output_simple(line.strip())
                     if filtered_line:
+                        # Add newline after dots if we've printed any
+                        if (
+                            self.dots_printed
+                            and hasattr(self, "original_stdout")
+                            and self.original_stdout
+                        ):
+                            self.original_stdout.write("\n")
```

Comment:
This ensures that the next real log message shows on a newline from the ...............